crates/rolldown/tests/esbuild/glob/tests.txt

```diff
@@ -0,0 +1,2 @@
+TestTSGlobBasicNoSplitting
```

Comment:
It's used to record what tests aren't migrated. I guess it's following what I did in https://github.com/rolldown/rolldown/pull/782.

Reply:
Yes，I followed this pr https://github.com/rolldown/rolldown/pull/782 to ensure the missing test case would be added in the future since I saw rolldown currently should not consider ts related test case.

Reply:
I didn't know file. I just take a look, it's created long time ago. I'm fine with unifying them. @IWANABETHATGUY Feel free to decide how to handle `ignored-test.md` and `tests.md`.

Reply:
Looks like the file is a little redundant? Since the `.` prefix is already means it is a ignored tests

Reply:
Nah. tests in `tests.md` means tests we have not copied/migrated from esbuild.

Reply:
It's a little bit chaos since now we have `tests.md`, `ignored-tests.txt` and adding `.`. That's why I support unifying them.

Reply:
got it